client/my-sites/domains/domain-management/name-servers/wpcom-nameservers-toggle.jsx

```diff
@@ -25,13 +26,18 @@ class NameserversToggle extends PureComponent {
 				</span>
 
 				<form className="name-servers__toggle">
-					<ToggleControl
-						id="wp-nameservers"
-						name="wp-nameservers"
-						onChange={ this.handleToggle }
-						checked={ this.props.enabled }
-						value="active"
-					/>
+					<FormLabel>
+						<ToggleControl
+							id="wp-nameservers"
+							name="wp-nameservers"
+							onChange={ this.handleToggle }
+							checked={ this.props.enabled }
+							value="active"
+						/>
+						<ScreenReaderText>
```

Comment:
The `ScreenReaderText` is a step forward for making this toggle more accessible 👍 

There are essentially 2 flaws I see here with this approach:

* We duplicate the `Use WordPress.com Name Servers` string. It would be better if we only had to include it just once in this component, and that would be sufficient for both the visible label and the screen readers.
* The visible label is still disconnected from the toggle. I'd expect that if I click the `Use WordPress.com Name Servers` label, I'd enable/disable the toggle. Note that this should also be evident when navigating with a screen reader.

Reply:
Both these points are very important

* I have now avoided the duplication of `Use WordPress.com Name Servers` string by using a constant, hope that is the correct approach. The same constant is used for both the visible label and the screen reader text
* Visible label is now connected to the toggle, if you click the visible label it will enable/disable toggle

Important:

* Removing the <FormLabel> for some reason is causing a situation where the ScreenReaderText is not working any more when testing with the voiceover tool in mac, any suggestions on this would be great. 🙏🏼